test/test_propagate_MonteCarlo.py

```diff
@@ -43,6 +43,29 @@
 # input signal + run methods
 x = rect(time, 100 * Ts, 250 * Ts, 1.0, noise=sigma_noise)
 
+# evaluate UMC_generic using the following (dummy) functions
+sample_shape = (2, 3, 4)
+draw_samples = lambda size: np.random.rand(size, *sample_shape)
+evaluate_sample = functools.partial(np.mean, axis=1)
+
+UMC_generic_multiprocess_kwargs = {
+    "draw_samples": draw_samples,
+    "evaluate": evaluate_sample,
+    "runs": 10,
+    "blocksize": 3,
+    "runs_init": 3,
+}
+
+UMC_generic_cov_kwargs = {
+    "draw_samples": draw_samples,
+    "evaluate": evaluate_sample,
+    "runs": 5,
+    "blocksize": 2,
+    "runs_init": 2,
+    "return_samples": True,
+    "return_histograms": False,
+}
+
```

Comment:
@mgrub Would it be okay for you, if I refactor those into fixtures to avoid global variables?

Reply:
Yes, thank you for the suggestion. That would be very good.

Reply:
Applied changes in [8368c69](https://github.com/PTB-M4D/PyDynamic/pull/306/commits/8368c69d5240d563c8cf6375aac282c8a30581be).